.github/workflows/continuous-integration.yml

```
@@ -135,6 +135,31 @@ jobs:
             extension: "pdo_pgsql"
             config-file-suffix: "-stringify_fetches"
 
+  phpunit-postgis:
+    name: "PHPUnit with PostGIS"
+    needs: "phpunit-smoke-check"
+    uses: ./.github/workflows/phpunit-postgis.yml
+    with:
+      php-version: ${{ matrix.php-version }}
+      postgis-version: ${{ matrix.postgis-version }}
+      extension: ${{ matrix.extension }}
+      postgres-locale-provider: ${{ matrix.postgres-locale-provider }}
+
+    strategy:
+      matrix:
+        php-version:
+          - "8.3"
+          - "8.4"
```

```suggestion
          - "8.5"
```

I don't expect any new insights from testing on multiple PHP versions, do you? Let's not bloat the matrix too much. Let's test with the lastest PHP version on both extensions and on the lowest supported PHP version with one extension only.

---

Applied as suggested.